runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/widgets/FieldTemplate/FieldTemplate.tsx

```diff
@@ -25,37 +27,72 @@ const FieldTemplate = ({
     rawHelp,
     rawDescription,
     schema,
-}: FieldTemplateProps) => (
-    <WrapIfAdditional
-        classNames={classNames}
-        disabled={disabled}
-        id={id}
-        label={label}
-        onDropPropertyClick={onDropPropertyClick}
-        onKeyChange={onKeyChange}
-        readonly={readonly}
-        required={required}
-        schema={schema}
-    >
-        <FormControl fullWidth error={!!rawErrors.length} required={required}>
-            {children}
-            {displayLabel && rawDescription ? (
-                <Typography variant="caption" color="textSecondary">
-                    {rawDescription}
-                </Typography>
-            ) : null}
-            {rawErrors.length > 0 && (
-                <List dense disablePadding>
-                    {rawErrors.map((error) => (
-                            <ListItem key={error} disableGutters>
+}: FieldTemplateProps) => {
+    const [rawDescriptionTranslatedIfKeyExists, setRawDescriptionTranslatedIfKeyExists] = useState(rawDescription);
```

Comment:
that's definetely too long, set it as rawDescription and rename prop. Btw making state on the base of prop is antipattern